analyzers/tests/SonarAnalyzer.UnitTest/Rules/Utilities/FileMetadataAnalyzerTest.cs

```diff
@@ -134,27 +136,37 @@ public void CreateMessage_NoEncoding_SetsEmptyString()
             var testRoot = Root + testName;
             Verifier.VerifyUtilityAnalyzer<FileMetadataInfo>(
                 projectFiles,
-                new TestFileMetadataAnalyzer(testRoot),
+                new TestFileMetadataAnalyzer(testRoot, true),
                 @$"{testRoot}\file-metadata.pb",
-                (messages) =>
-                {
-                    for (var i = 0; i < autogeneratedFiles.Length; i++)
-                    {
-                        var message = messages[i];
-                        message.IsGenerated.Should().BeTrue();
-                        message.FilePath.Should().Be(autogeneratedFiles[i]);
-                    }
-                },
+                VerifyProtoBuf,
+                parseOptions);
+
+            Verifier.VerifyUtilityAnalyzer<FileMetadataInfo>(
+                projectFiles,
+                new TestFileMetadataAnalyzer(testRoot, false),
+                @$"{testRoot}\file-metadata.pb",
+                VerifyProtoBuf,
```

Comment:
These lines seem to be the same, except `true/false`, so it can be a local method, to make it easier to read